tests/Repository/AamcMethodRepositoryTest.php

```diff
@@ -28,10 +27,8 @@ protected function setUp(): void
         ]);
         $this->fixtures = $executor->getReferenceRepository();
 
-        $this->repository = self::$kernel->getContainer()
-            ->get('doctrine')
-            ->getManager()
-            ->getRepository(AamcMethod::class);
+        $entityManager = self::$kernel->getContainer()->get('doctrine')->getManager();
+        $this->repository = $entityManager->getRepository(AamcMethod::class); /** @phpstan-ignore-line */
```

Comment:
I think I'm really going to miss the thing ember has where you can cool down a warning for a few weeks and then get back to it. Anyway to annotate this and remember to align the expectation in the test with the code? Maybe a `@todo` at least?

Reply:
Never mind, cleaned up in another way.